components/core/DownloadManager.py

```diff
@@ -36,15 +36,15 @@ def remove_download(id, userName):
     db = threadpool.connect()
     if db is not None:
         cursor = db.cursor()
-        sql1 = "SELECT status FROM download WHERE id=%s;"
+        sql1 ="SELECT status FROM download WHERE id=%s;"
         sql = "DELETE from download WHERE id=%s and user_name=%s;"
         try:
-            cursor.execute(sql1, id)
+            cursor.execute(sql1,str(id))
```

Comment:
If the value of id exceeds 10 , delete fails. Please suggest something regarding this.